nightingale_interface/screens/admin_screen.py

```diff
@@ -0,0 +1,89 @@
+from kivy.uix.screenmanager import Screen
+from kivy.uix.image import Image
+
+from kivymd.uix.label import MDLabel
+from kivymd.uix.button import MDRectangleFlatButton
+from kivy.uix.button import Button
+
+from kivy.uix.screenmanager import SlideTransition, NoTransition
+
+from screens.screen_config import ScreenConfig as cfg
+
+
+class AdminScreen:
+    def estop(self, button_data):
+        button_data.parent.manager.transition = NoTransition()
+        cfg.LAST_SCREEN = "adminscreen"
+        button_data.parent.manager.current = "estoppedscreen"
+
+    def to_home(self, button_data):
+        button_data.parent.manager.transition = NoTransition()
+        cfg.LAST_SCREEN = "adminscreen"
+        button_data.parent.manager.current = "homescreen"
+
+    def shutdown(self, button_data):
+        # shutdown system, might not be useful since HW button exists
+        pass
+
+    def release_bin(self, button_data):
```

Comment:
Ben I was wrong and here are some more places in which we can add the ROS topic publishing besides the confirmation file. We could hook these up the the confirmation screen if we really want to put all ROS topic publishing there

Reply:
I feel like for these actions confirmation is not as important since it would only tell the robot to open and close the grippers which is not critical